.changeset/every-actors-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@reown/appkit-scaffold-ui': patch
+---
+
+disable `open` button until connection established
```

Comment:
We generally follow a practice where explanations start with capital letters unlike the commits for changeset summary